### DISCUSSION TITLE
Is there a way to get all of the available variables generated by JIT?

### DISCUSSION THREAD

**Discussion by ericlunadev** [tailwindlabs/tailwindcss#8336] [Help]
I'm trying to add an autocomplete feature to a Tailwind class input and I was wondering if there is a way to get all of the possible classes that JIT can generate, including the ones when you modify the config.

Thanks!

**Comment by await-ovo** [comment]
You can get the full class list with `context.getClassList()`, which is what I did in [tailwindcss-webpack-plugin](https://github.com/await-ovo/tailwindcss-webpack-plugin/blob/main/src/util.ts#L161).
